index.html

```diff
@@ -42,5 +42,177 @@ <h1 class="name"> Hey there <br>
           <img src="images/Disabled.svg" alt="drop-down">
         </div>
     </section>
+    <section class="line-break">
+            <h2 class="works">My Recent <br> Works</h2>
+    </section>
+    
+      <section class="work-section">
+        <div  class="pictureholder-container">
+            <img src="Images 2/placeholder.svg" alt="placeholder" class="placeholder">
+            <div class="text-container">
+              <h5  class="header2">Multi-Post Stories <br> Gain+Glory</h5>
+              <ul class="text">
+                <li class="text-ruby">Ruby on Rails</li>
+                <li class="text-css">CSS</li>
+                <li class="text-java">JavaScript</li>
+                <li class="text-html">HTML</li>
+              </ul>
```

Comment:
Great job on adding the technologies list as requested by the previous reviewer! However, there are a few adjustments that need to be made to align it with the design.

- [ ] The padding for the technologies list should be 16px as per the design, not 20px. Please update the padding to 16px for all sides to match the design specifications. :pray: